### PR TITLE
Add notification to config/credential files pointing to help page

### PR DESCRIPTION
**Pull request by abrooksv**
## Screenshots (if appropriate)
Screen Shot 2021-06-02 at 12 23 56 PM

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] Local run of `gradlew check` succeeds
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import software.amazon.awssdk.profiles.ProfileFileLocation
+import software.aws.toolkits.jetbrains.core.credentials.CredentialFileHelpNotifications.CredentialFileNotificationPanel
+import software.aws.toolkits.jetbrains.core.help.HelpIds
+import software.aws.toolkits.resources.message
+
+class CredentialFileHelpNotifications : EditorNotifications.Provider<CredentialFileNotificationPanel>(), DumbAware {
+    override fun getKey(): Key<CredentialFileNotificationPanel> = KEY
+
+    override fun createNotificationPanel(file: VirtualFile, fileEditor: FileEditor, project: Project): CredentialFileNotificationPanel? {
+        if (PropertiesComponent.getInstance().isTrueValue(DISABLE_KEY)) return null
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt:24]: makes sense if you read the rest of the file but probably worth a comment explaining these conditions

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import com.intellij.openapi.util.Key
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.ui.EditorNotificationPanel
+import com.intellij.ui.EditorNotifications
+import software.amazon.awssdk.profiles.ProfileFileLocation
+import software.aws.toolkits.jetbrains.core.credentials.CredentialFileHelpNotifications.CredentialFileNotificationPanel
+import software.aws.toolkits.jetbrains.core.help.HelpIds
+import software.aws.toolkits.resources.message
+
+class CredentialFileHelpNotifications : EditorNotifications.Provider<CredentialFileNotificationPanel>(), DumbAware {
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt:20]: nit: ‘CredentialsFileHelpNotificationProvider’?

**Review by rli — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt (hunk 2 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt]: Changed

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt]: Added

File: resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 3]
```diff
@@ -520,9 +521,12 @@ general.execution.failed=failed
 general.execution.running=running...
 general.execution.success=completed successfully
 general.file_not_found=File not found: "{0}"
+general.help=Help
 general.in_progress_button=In progress
 general.message=Message
 general.name.label=Name:
+general.notification.action.hide_once=Dismiss
+general.notification.action.hide_forever=Don't show again
```
Comment (manodnyab) [inline resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:529]: Possible duplicate of `notice.suppress`, which is pretty much the same message

Comment (abrooksv) [reply resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:529]: Removed notice.suppress

**Review by manodnyab — APPROVED** [review]

**Pull request merged**